apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-allowed-users/facility-allowed-users.component.ts

```diff
@@ -156,47 +154,45 @@ export class FacilityAllowedUsersComponent implements OnInit {
 
   voSelected(vo: Vo): void {
     // prevents "fake" triggers
-    if (this.selectedVo.id === vo.id) {
+    if (this.selectedVo !== undefined && this.selectedVo.id === vo.id) {
       return;
     }
```

Comment:
I believe we fixed the select component to not emit if the same object is selected. Could you investigate if this check is still needed (same for resources) ?